cmd/fb-om-exporter/main.go

```diff
@@ -136,9 +144,15 @@ func main() {
 		http.HandleFunc("/metrics/array", func(w http.ResponseWriter, r *http.Request) {
 			metricsHandler(w, r)
 		})
+		http.HandleFunc("/metrics/buckets", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
consider renaming to /metrics/objectstore and including purefa_object_store_account metrics as well in the same uri .

Reply:
Completed in 33abc37dc5707b50af1366c5930ab5313af50bff